guide/installation/gmc_install/gmc_install.md

```diff
@@ -0,0 +1,118 @@
+# GenAI-microservices-connector(GMC) Installation
+
+This document will introduce the GenAI Microservices Connector (GMC) and its installation. It will then use the ChatQnA pipeline as a use case to demonstrate GMC's functionalities.
+
+## GenAI-microservices-connector(GMC)
+
+GMC can be used to compose and adjust GenAI pipelines dynamically on Kubernetes. It can leverage the microservices provided by GenAIComps and external services to compose GenAI pipelines. External services might be running in a public cloud or on-prem by providing an URL and access details such as an API key and ensuring there is network connectivity. It also allows users to adjust the pipeline on the fly like switching to a different Large language Model(LLM), adding new functions into the chain(like adding guardrails),etc. GMC supports different types of steps in the pipeline, like sequential, parallel and conditional. For more information: https://github.com/opea-project/GenAIInfra/tree/main/microservices-connector
+
+
+
+## Install GMC
+
+**Prerequisites**
+
+- A running Kubernetes cluster with at least 16 CPUs, 32GB of memory, and 100GB of disk space. To install an Kubernetes cluster as your platform:
```

Comment:
Perhaps say "For the ChatQnA example ensure your cluster has .." (Don't know what the others need....)